### ISSUE TITLE
Cannot use other template engines since version 3.0

### ISSUE THREAD

**Issue by ryankask** [incuna/django-wkhtmltopdf#101]
In version 3.0, a lot of code was refactored, including ``render_to_temporary_file()`` (see PR #95). In that function, ``context`` is now converted into a ``Context`` object, which is a class related to Django's template language.

Django 1.8 introduced pluggable template engines and I had no problems using Jinja2 with version 2.0.3 of django-wkhtmltopdf.

This new code in ``render_to_temporary_file()`` causes problems with other template engines:

```python
# make sure the context is a context object
if not isinstance(context, (Context, RequestContext)):
    context = Context(context)

content = smart_text(template.render(context))
```

The ``render()`` method, according to [DEP 182](https://github.com/django/deps/blob/master/final/0182-multiple-template-engines.rst#backends-api), is actually supposed to take a ``dict``. The DTL backend's ``Template`` class handles converting the ``dict`` to a ``Context`` or ``RequestContext``.

Can this template engine dependency be removed?

I noticed issue #100 discusses warnings that are the result of the some of the changes above. I raised a separate issue since 3.0 is no longer usable for users of other template engines.

Thanks for your help.

**Comment by johnraz** [comment]
Hi!
Seems reasonable, do you mind submitting a PR ?

**Comment by johnraz** [comment]
Closing as this was fixed by #102
